src/Tecan/BasicCommands/Dispense.php

```diff
@@ -2,12 +2,12 @@
 
 namespace Mll\LiquidHandlingRobotics\Tecan\BasicCommands;
 
-use Mll\LiquidHandlingRobotics\Tecan\LiquidClass;
+use Mll\LiquidHandlingRobotics\Tecan\LiquidClass\LiquidClass;
 use Mll\LiquidHandlingRobotics\Tecan\Location\Location;
 
 final class Dispense extends BasicPipettingActionCommand implements Command
 {
-    public function __construct(int $volume, Location $location, LiquidClass $liquidClass)
+    public function __construct(float $volume, Location $location, LiquidClass $liquidClass)
```

Comment:
Es werden keine Volumina mit Nachkommastellen Pipettiert. Daher int

Reply:
The current methods for cnv uses float. Thus I'd keep it and added the comment from the manual:

> Floating point values are also accepted and do not cause an error, but they will be
rounded before being used. In such cases, it is recommended to use integer
calculations to avoid unexpected results.